data/root/hostip_from_wicked

```diff
@@ -1,5 +1,17 @@
 #! /usr/bin/perl
 
+# helper script that
+#
+# (1) writes a list of assigned IP adresses to ARG1, and
+#
+# (2) writes the FQDN or, if not available, the first IP to ARG2
+#
+# The address in (2) should be something we can be reached by via network.
+#
+# usage:
+#
+# hostip_from_wicked ARG1 ARG2
+
```

Comment:
Thanks for the docs!

I would also add a sample wicked output, so that the parser loop makes more sense to the future maintainer:
```console
# wicked show all
lo              up
      link:     #1, state up
      type:     loopback
      config:   compat:suse:/etc/sysconfig/network/ifcfg-lo
      leases:   ipv4 static granted
      leases:   ipv6 static granted
      addr:     ipv4 127.0.0.1/8 [static]
      addr:     ipv6 ::1/128 [static]

ens7            up
      link:     #2, state up, mtu 1500
      type:     ethernet, hwaddr 52:54:00:05:0a:2c
      config:   compat:suse:/etc/sysconfig/network/ifcfg-ens7

eth1            up
      link:     #3, state up, mtu 1500
      type:     ethernet, hwaddr 52:54:00:d7:57:86
      config:   compat:suse:/etc/sysconfig/network/ifcfg-eth1
      leases:   ipv4 dhcp granted
      leases:   ipv6 dhcp requesting
      addr:     ipv4 10.230.84.112/18 [dhcp]
      route:    ipv4 default via 10.230.63.254
```